Package.swift

```diff
@@ -0,0 +1,21 @@
+// swift-tools-version:5.1
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "TICodeView",
```

Comment:
Usually, it's called One Time Pin or One Time Password - OTP. You can name it like OTPSwiftView or something like that (OTPView is taken by few android libraries, but you can still take it)